contributing/documentation/overview.rst

```diff
@@ -15,15 +16,38 @@ Before Your First Contribution
 
 **Before contributing**, you should consider the following:
 
-* Symfony documentation is written using reStructuredText_ markup language.
-  If you are not familiar with this format, read :doc:`this article </contributing/documentation/format>`
-  for a quick overview of its basic features.
-* Symfony documentation is hosted on GitHub_. You'll need a GitHub user account
-  to contribute to the documentation.
+* Symfony documentation is written using `reStructuredText`_ markup language.
+  If you are not familiar with this format, read
+  :doc:`this article </contributing/documentation/format>` for a quick overview
+  of its basic features.
+* Symfony documentation is hosted on `GitHub`_. You'll need a free GitHub user
+  account to contribute to the documentation.
 * Symfony documentation is published under a
   :doc:`Creative Commons BY-SA 3.0 License </contributing/documentation/license>`
   and all your contributions will implicitly adhere to that license.
 
+.. _minor-changes-e-g-typos:
+
+Fast Online Contributions
+-------------------------
+
+This method is best suited for small contributions, such as typos and rewordings.
+Due to GitHub's functional frontend, it is possible to create Pull Requests
+right in your browser while reading the docs on symfony.com.
+
+First, switch to the oldest maintained Symfony branch which contains the error,
+which is usually ``2.3``. Then, click on the **edit this page** button on the
+upper right corner. Now edit the content and describe your changes within the
+GitHub frontend.
```

Comment:
To be clear, I mean remove the first sentence only (then re-word the second)

A picture here would be sweet too.